umh.docs.umh.app/content/en/docs/concepts/security/management-console.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Management Console"
+content_type: concept
+description: |
+    Concepts related to the security of the Management Console.
+weight: 10
+---
+
```

Comment:
We should explain here in detail how we are planning to encrypt it, and how it works. Also we should make clear how the user can communicate via our cloud, but we have no technical possibility to read the exchanged data (like we had in the previous security page on mgmt.docs.umh.app)